Tests/js/views/universaldiscovery/assets/ez-universaldiscoveryfinderexplorerview-tests.js

```diff
@@ -153,6 +153,17 @@ YUI.add('ez-universaldiscoveryfinderexplorerview-tests', function (Y) {
                 );
             }, this));
         },
+
+        "Should set ownSelectedItem attribute to false when the explorer view wakeUp": function () {
+            Y.Array.each(this.levelViews, Y.bind(function (levelView) {
```

Comment:
that's a bit weird. I think you should just call `this.view.wakeUp()` and check the `ownSelectedItem` attribute value is `false`